### ISSUE TITLE
Adapt the RESTful API

### ISSUE THREAD

**Issue by renanrodrigo** [kytos/mef_eline#8]
To create, list and remove EVCs.

**Comment by viniciusarcanjo** [comment]
This is partially completed. The methods POST and GET, to create, list and get a specific EVC id, are already in place on PR #27 / PR #23 .

**Comment by viniciusarcanjo** [comment]
Since @macartur is working on most fixes of mef_eline at the moment, I'll remove myself as the assignee. @beraldoleal and @macartur I believe we should try to add the remove methods on EVCs and on the REST API on this beta3 release. Without EVC DELETE it's been really difficult to test it at AmLight. For example, the other day I pushed a few EVCs, and I had to remove everything, including other flows, since removing one by one is not practical (from a trial/testing lab perspective).
